mcs/class/corlib/System.IO/DriveInfo.cs

```diff
@@ -55,15 +55,17 @@ public DriveInfo (string driveName)
 			}
 
 			DriveInfo [] drives = GetDrives ();
+			Array.Sort (drives, (DriveInfo di1, DriveInfo di2) => String.Compare (di2.path, di1.path, true));
+			var sb = new StringBuilder($"driveName: {driveName}; available drives: ");
```

Comment:
I don't think it's good idea to expose this in the exception

Reply:
Oh, it's just for small diagnostic. I'll remove it.